lib/browser.ts

```diff
@@ -114,15 +120,15 @@ function buildElementHelper(browser: ProtractorBrowser): ElementHelper {
 /**
  * @alias browser
  * @constructor
- * @extends {webdriver.WebDriver}
+ * @extends {webdriver_extensions.ExtendedWebDriver}
```

Comment:
In order for js doc to work, this is the correct type (see old version pointing to `webdriver.WebDriver` not `Webdriver`)